doc/nrf/libraries/caf/ble_adv.rst

```diff
@@ -106,6 +113,7 @@ The module automatically gets new advertising data and scan response data from B
 
 * Bluetooth LE undirected advertising is started or restarted.
 * Undirected advertising enters the :ref:`caf_ble_adv_grace_period`.
+* Resolvable Private Address (RPA) is rotated with the :kconfig:option:`CONFIG_CAF_BLE_ADV_ROTATE_RPA` option enabled.
```

Comment:
```suggestion
* RPA is rotated with the :kconfig:option:`CONFIG_CAF_BLE_ADV_ROTATE_RPA` option enabled.
```
We expand the abbreviations only in the 1st instance and then we could use the abbreviation in the rest places of the document.